--- No Good Gofers (Williams 1997) V1.1.2/No Good Gofers (Williams 1997) V1.1.2.vbs.original	2026-04-03 19:42:39
+++ No Good Gofers (Williams 1997) V1.1.2/No Good Gofers (Williams 1997) V1.1.2.vbs	2026-04-03 19:41:05
@@ -320,9 +320,9 @@
     White_Pumper.visible = visState
     
     ' Objects WITH spaces - use square brackets
-    [L178 Side Flasher].visible = visState
-    [L180 Side Flasher].visible = visState
-    [L179 Side Flasher].visible = visState
+    GetElementByName("L178 Side Flasher").visible = visState
+    GetElementByName("L180 Side Flasher").visible = visState
+    GetElementByName("L179 Side Flasher").visible = visState
     
     On Error Goto 0
 End Sub
